src/sass/components/_feedback.scss

```diff
@@ -0,0 +1,88 @@
+.section--feedback {
+  padding: 50px 0 25px;
+
+  .panel-bar {
+    .heading {
+      h5 {
+        color: #d58e32;
+        text-transform: uppercase;
```

Comment:
dodaj zmienną, zamiast koloru na sztywno w kodzie

uppercase - pytanie czy klient sam powinien wprowadzać tekst drukowanymi kiedy chce, czy jest to ustawione na sztywno w kodzie

Reply:
Uppercase jest w kodzie, tak są stylowane także inne tego typu elementy

Reply:
Zmienna dodana